app/(admin)/openmailer/campaigns/[id]/CampaignEditForm.tsx

```
@@ -0,0 +1,253 @@
+    bodyHtml: campaign.bodyHtml,
+    bodyText: campaign.bodyText ?? '',
+    listName: campaign.listName,
+  });
+
+  const isPlain = campaign.emailFormat === 'plain';
+  const canEdit = campaign.status === 'draft' || campaign.status === 'scheduled';
+
+  async function handleSave() {
+    const isPlain = campaign.emailFormat === 'plain';
+    if (isPlain && !form.bodyText.trim()) {
+      alert('Enter the plain text email body.');
```

ESLint

[error] Unexpected alert.

(no-alert)